x/axelarnet/keeper/grpc_query.go

```diff
@@ -38,8 +39,18 @@ func (q Querier) PendingIBCTransferCount(c context.Context, _ *types.PendingIBCT
 		if !ok {
 			return nil, fmt.Errorf("cosmos chain %s not found in the %s module", c, nexusTypes.ModuleName)
 		}
-		transfers := q.nexus.GetTransfersForChain(ctx, chain, exported.Pending)
-		counts[c.String()] = uint32(len(transfers)) // assert: there should never be more than 4294967295 transfers in the queue
+		pageRequest := &query.PageRequest{
+			Key:        nil,
+			Offset:     0,
+			Limit:      1,
```

Comment:
why limit is 1?

Reply:
It was just a dummy arg, since we don't care about the transfers itself, just the total. Looking into the code, I could also pass in `0` but that would end up using the default of loading `100` [transfers](https://github.com/cosmos/cosmos-sdk/blob/v0.45.6/types/query/pagination.go#L69) lol